worlds/osrs/docs/en_Old School Runescape.md

```diff
@@ -0,0 +1,21 @@
+# Old SChool Runescape
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+
```

Comment:
Something tells me this isn't quite done :D

Reply:
Yeah, I have held off on the user-facing documentation while I wait for Runelite approval so I can include more info about how the end-user will actually install and use the plugin.